base-template.sh

```diff
@@ -527,10 +572,23 @@ update_shared_hooks_if_appropriate() {
 
         for SHARED_REPO in $SHARED_REPOS_LIST; do
             unset IFS
-            mkdir -p "$INSTALL_DIR/shared"
 
             set_shared_root "$SHARED_REPO"
 
+            if [ "$SHARED_REPO_IS_CLONED" != "true" ]; then
+                # Non-cloned roots are ignored
+                continue
+            elif [ "$SHARED_HOOKS_TYPE" = "--local" ] &&
+                [ "$SHARED_REPO_IS_LOCAL" = "true" ]; then
+                echo "! Warning: Local shared hooks contain an url" >&2
```

Comment:
Please use `URL` everywhere instead of `url`.
Also I believe it's `a URL` rather than `an URL`, please. :bow: